cpp/tests/lists/contains_tests.cpp

```diff
@@ -108,11 +95,30 @@ auto create_null_search_key()
 
 }  // namespace
 
+auto constexpr X          = int32_t{0};     // Placeholder for nulls.
+auto constexpr ABSENT     = size_type{-1};  // Index when key is not found in a list.
+auto constexpr FIND_FIRST = lists::duplicate_find_option::FIND_FIRST;
+auto constexpr FIND_LAST  = lists::duplicate_find_option::FIND_LAST;
+
+using bools_col   = cudf::test::fixed_width_column_wrapper<bool, int32_t>;
+using int32s_col  = cudf::test::fixed_width_column_wrapper<int32_t>;
```

Comment:
For the return values in these tests, we should check against values of `size_type` instead of hardcoding the present definition that `size_type = int32_t`.